docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -1,3 +1,4 @@
+@![iOS]
```

Comment:
This part of the header is generic:
>Adding the Lock Screen

>The lock screen is a vital part of SmartDeviceLink, as the lock screen prevents the user from using your application while the vehicle is in motion. SDL takes care of the lock screen for you. It still allows you to use your own view controller if you prefer your own look, but still want the recommended logic that SDL provides for free.
A benefit to using the provided Lock Screen is that we also handle retrieving a lock screen icon for versions of Core that support it, so that you do not have to be concerned with what car manufacturer you are connected to. If you subclass the provided lock screen, you can do the same in your own lock screen view controller.

This part of the header is iOS only:
>If you would not like to use any of the following code, you may use the SDLLockScreenConfiguration class function disabledConfiguration, and manage the entire lifecycle of the lock screen yourself. However, it is strongly recommended that you use the provided lock screen manager, even if you use your own view controller.
To see where the SDLLockScreenConfiguration is used, refer to the Integration Basics guide.

Reply:
Also, the lock screen is only part of the iOS and Android libraries. JavaSE and JavaEE can be omitted in the config file.

Reply:
fixed